src/main/java/MovieUI/MovieHandler.java

```
@@ -0,0 +1,723 @@
+            AnchorPane posterView = loader.load();
+            InfoController controller = loader.getController();
+
+            controller.getMovieTitleLabel().setText(movie.getTitle());
+            controller.getMovieRatingLabel().setText(String.format("%.2f", movie.getRating()));
+            if (movie.getReleaseDate() != null) {
+                Date date = movie.getReleaseDate();
+                Calendar calendar = Calendar.getInstance();
+                calendar.setTime(date);
+                String printDate = new SimpleDateFormat("dd/MM/yyyy").format(date);
+                controller.getMovieReleaseDateLabel().setText(printDate);
+            } else {
+                controller.getMovieReleaseDateLabel().setText("N/A");
+            }
+            try {
+                Image posterImage = new Image(movie.getFullBackdropPath(), true);
+                controller.getMovieBackdropImageView().setImage(posterImage);
+            } catch (NullPointerException ex) {
+
+            }
+
+            controller.getMovieSummaryLabel().setText(movie.getSummary());
+            controller.getMovieCastLabel().setText(RetrieveRequest.getCastStrings(movie));
+            controller.getMovieCertLabel().setText(RetrieveRequest.getCertStrings(movie));
+            String[] genres = RetrieveRequest.getGenreStrings(movie);
+            StringBuilder builder = new StringBuilder();
+            try {
+                for (String genre : genres) {
+                    builder.append(genre);
+                    System.out.println(genre + "  " + genres.length);
+                    // if not last string in array, append a ,
+                    if (genres.length == 0) {
+                        System.out.println("no genres");
+                    } else if (!genres[genres.length - 1].equals(genre)) {
+                        builder.append(", ");
+                    }
+                }
+            } catch (NullPointerException ex) {
+
```

Why empty catch block?

If it needs to be empty for some reason, provide a reason for it.